python/src/LinearModelAnalysis.i

```diff
@@ -9,3 +9,129 @@
 %copyctor OT::LinearModelAnalysis;
 
 %include openturns/LinearModelAnalysis.hxx
+namespace OT { %extend LinearModelAnalysis { LinearModelAnalysis(const LinearModelAnalysis & other) { return new OT::LinearModelAnalysis(other); } } }
```

Comment:
you can use "%copyctor OT::LinearModelAnalysis;" instead

Reply:
Can you be more specific? Please suggest the exact change to the code, as I do not fully understand what it means.

Reply:
indeed, then line 12 can be removed